pkg/crc/version/version.go

```diff
@@ -88,22 +92,22 @@ func getCRCLatestVersionFromMirror() (*semver.Version, error) {
 		return nil, err
 	}
 	defer response.Body.Close()
+
 	if response.StatusCode != 200 {
 		return nil, fmt.Errorf("HTTP error: %s: %d", response.Status, response.StatusCode)
 	}
+
 	releaseMetaData, err := ioutil.ReadAll(response.Body)
 	if err != nil {
 		return nil, err
 	}
-	err = json.Unmarshal(releaseMetaData, &releaseInfo)
-	if err != nil {
+
+	var releaseInfo CrcReleaseInfo
+	if err := json.Unmarshal(releaseMetaData, &releaseInfo); err != nil {
 		return nil, fmt.Errorf("Error unmarshaling JSON metadata: %v", err)
 	}
-	version, err := semver.NewVersion(releaseInfo.Version.LatestVersion)
-	if err != nil {
-		return nil, err
-	}
-	return version, nil
+
+	return releaseInfo.Version.CrcVersion, nil
```

Comment:
Might be good to return an error if `Version` or `CrcVersion` are missing.

Reply:
Yes added a nil check, good point!

When the json was empty:
```
$ crc version
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x773e7a]

goroutine 1 [running]:
github.com/Masterminds/semver/v3.(*Version).Compare(0x0, 0xc0008f60f0, 0xc0008f60f0)
	/home/guillaumerose/gomod/crc/vendor/github.com/Masterminds/semver/v3/version.go:388 +0x3a
github.com/Masterminds/semver/v3.(*Version).GreaterThan(...)
	/home/guillaumerose/gomod/crc/vendor/github.com/Masterminds/semver/v3/version.go:368
github.com/code-ready/crc/cmd/crc/cmd.newVersionAvailable(0x1b2991c, 0xc00051bb78, 0x8017bc, 0x1815d60, 0x2be5540, 0x1815d60, 0x2be5540)
	/home/guillaumerose/gomod/crc/cmd/crc/cmd/start.go:222 +0x9a
github.com/code-ready/crc/cmd/crc/cmd.checkIfNewVersionAvailable(0x1815d00, 0x2be5540, 0x0)
	/home/guillaumerose/gomod/crc/cmd/crc/cmd/start.go:201 +0x44
```